gdal/doc/source/drivers/raster/msg.rst

```diff
@@ -40,12 +35,7 @@ Driver capabilities
 Build Instructions
 ------------------
 
-Download the Eumetsat library for wavelet decompression. This is a file
-named PublicDecompWT.zip. Extract the content in a subdirectory with the
-same name (under frmts/msg).
-
-If you are building with Visual Studio 6.0, extract the .vc makefiles
```

Comment:
This part about .vc files is probably still needed. GDAL's frmts/msg/makefile.vc expects makefile.vc files in PublicDecompWT, not the Makefile files (now?) provided by https://gitlab.eumetsat.int/open-source/PublicDecompWT. That said, I've never tested myself Windows builds with this driver

Reply:
Why did you mark this comment as resolved ? The part about extracting the .vc makefiles is still deleted

Reply:
Sorry, it looks like my comment that I thought I'd submitted somehow isn't here. We no longer offer the VC files - they were outdated, untested, and probably incomplete. Support for Cygwin is there but no Visual Studio.

Reply:
> We no longer offer the VC files

ok, but GDAL git does contains a zip with makefile.vc files: https://github.com/OSGeo/gdal/blob/master/gdal/frmts/msg/PublicDecompWTMakefiles.zip. This is what was mentionned here. No idea if it is still functional, but in doubt, I'd let the old comment, perhaps with a note that it hasn't been tested lately